apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -311,7 +310,7 @@ def from_h5ad(
     uns_keys: Sequence[str] | None = None,
     additional_metadata: AdditionalMetadata = None,
 ) -> str:
-    """Reads an ``.h5ad`` file and writes it to an :class:`Experiment`.
+    r"""Reads an ``.h5ad`` file and writes it to an :class:`Experiment`.
```

Comment:
```suggestion
    """Reads an ``.h5ad`` file and writes it to an :class:`Experiment`.
```

Reply:
it is correct as is (won't pass lint otherwise).

the reason is the slashes in the docstring